lib/authentication.js

```diff
@@ -1,17 +1,27 @@
-
 // core dependencies
+const path = require('path')
 const url = require('url')
 
+// npm dependencies
+const slash = require('slash')
+
 // local dependencies
 const config = require('./config')
+const plugins = require('./plugins/plugins')
 const { encryptPassword } = require('./utils')
 
+// Locate GOV.UK Frontend paths
+const { baseDir, entryPath } = plugins.govukFrontendPaths()
+
 // Local variables
 const allowedPathsWhenUnauthenticated = [
   '/manage-prototype/password',
   '/public/stylesheets/unbranded.css',
   '/plugin-assets/govuk-prototype-kit/lib/assets/images/unbranded.ico',
-  '/plugin-assets/govuk-frontend/govuk/all.js',
+
+  // Allow GOV.UK Frontend package export only
+  `/plugin-assets/govuk-frontend/${slash(path.relative(baseDir, entryPath))}`,
```

Comment:
I've removed this line as we trust `/manage-prototype/dependencies/*` since [`d262239`](https://github.com/alphagov/govuk-prototype-kit/commit/d262239d601e92cc7af138fcf94c7a26172e6bd2#diff-984bd1a73f60669a089ccb687efdd328346bf021cd43567d7f79808c073241c7R40)